README.md

```diff
@@ -214,12 +214,14 @@ require("telescope").extensions.rest.select_env()
 - env_pattern: For env file pattern
 - env_edit_command: For env file edit command
 
+If running Ubuntu or Debian based systems you might need to run `ln -s $(which fdfind) ~/.local/bin/fd` to get extension to work. This becuase extension runs the [fd](https://github.com/sharkdp/fd?tab=readme-ov-file#installation) command.
+
 ## Lualine
 
 We also have lualine component to get what env file you select!
 And dont't worry, it will only show up under http files.
 
-```lua
+:x
```

Comment:
I don't think this change should be here

```suggestion
```lua
```

Reply:
fixed